Kurzanleitungen/3D-Drucker/3D-Drucker.tex

```diff
@@ -92,16 +92,16 @@ \section{Technische Daten}
 \newpage
 \section{Bedienung}
 \subsection{Allgemeines}
-Falls es kälter als 15 oder wärmer als 35 °C ist, direkte \textbf{Sonneneinstrahlung} auf den Drucker nicht vermeidbar ist oder sehr \textbf{staubige Arbeiten} im Raum durchgeführt werden, darf nicht gedruckt werden.\\
-Der 3D-Drucker und sein Tisch dürfen in keinem Fall bewegt werden, da sonst eine vollständige Kalibrierung von Nöten ist.\\
+Falls es kälter als 15°C oder wärmer als 35 °C ist, direkte \textbf{Sonneneinstrahlung} auf den Drucker nicht vermeidbar ist oder sehr \textbf{staubige Arbeiten} im Raum durchgeführt werden, darf nicht gedruckt werden.\\
+Der 3D-Drucker und sein Tisch dürfen \textbf{in keinem Fall} bewegt werden, da sonst eine vollständige Kalibrierung von Nöten ist.\\
 Die offizielle Anleitung (Englisch) findest du im Hängeregister "`3D-Drucker"' im Rollcontainer.
 \subsection{Druckvorbereitung}
 \begin{itemize}
 	\item Die 3D-Modelle müssen im '.stl'- oder '.obj'-Format vorliegen.
-	\item Das ‘Slicen’, also dem Erstellen der für 3D-Drucker lesbaren '.gcode',  geschieht mit der Software 'BCN3D Cura' und dem Custom-Profil 'fast with brim' oder 'standart with brim'. Die Profile sind auf dem Werkstatt-Github Repo\footnote{\url{https://github.com/fsi-tue/Werkstatt}} zu finden.
+	\item Das ‘Slicen’, also dem Erstellen der für den 3D-Drucker lesbaren '.gcode',  ge­schieht mit der Software 'BCN3D Cura' und dem Custom-Profil 'fast with brim' oder 'standart with brim'. Die Profile sind auf dem Werkstatt-Github Repo\footnote{\url{https://github.com/fsi-tue/Werkstatt}} zu finden.
```

Comment:
Interesting, that diff seems to be broken when viewed through the GH web UI...
Locally, Git shows me the following:
![image](https://user-images.githubusercontent.com/7537109/135716051-a01fe055-31e5-4f3c-b9a2-b5767ae2afca.png)

However, apparently my Chromium doesn't even render that hyphen/dash(?) when I paste it :roll_eyes:.
But I guess/hope it doesn't really matter anyway so I'll simply drop it.